package.json

```diff
@@ -3,15 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "firebase": "^8.2.1",
-    "husky": "^4.3.6",
+    "date-fns": "^2.16.1",
+    "firebase": "^8.2.4",
+    "husky": "^4.3.8",
     "lint-staged": "^10.5.3",
     "prettier": "^2.2.1",
-    "react": "^16.14.0",
-    "react-dom": "^16.14.0",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
     "react-firebase-hooks": "^2.2.0",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "^3.4.4"
+    "react-scripts": "^4.0.1"
```

Comment:
we don't need to update all these other packages if all you need is date-fns. We can take a look at how you're installing new packages so that you don't update other packages but in the meantime, you can change the file manually to ensure that those other packages don't get updated. Updating packages without investigating the changes is risky, you could easily crash your app so please watch out for this

Reply:
Thank for the advice. Also, the file was reversed.